Source/core/ResourceMonitor.h

```diff
@@ -20,6 +20,7 @@
 #pragma once
 
 #include "Module.h"
+#include <memory>
```

Comment:
just realized this is a strange place to put this; it should be with other standard headers if there are any. This appears to be an edge case where there aren't any other standard ones. (It cannot go below inside the ifdef.)